<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>461</width>
     <height>671</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QPushButton" name="origin_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>恢复原图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="image_enhancement">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>图像增强</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>保存当前图像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>图像分割</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sharpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>锐化</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>K:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
   <widget class="QWidget" name="page_3"/>
  </widget>
  <widget class="QLabel" name="Image_show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="image_path">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>630</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>图像文件：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Image_Choose_Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>630</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图像</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
